src/liboslexec/llvm_util.cpp

```diff
@@ -1591,8 +1582,14 @@ LLVM_Util::ptx_compile_group (llvm::Module* lib_module, const std::string& name,
     llvm::raw_svector_ostream assembly_stream (assembly);
 
     // TODO: Make sure rounding modes, etc., are set correctly
+#if OSL_LLVM_VERSION >= 70
     target_machine->addPassesToEmitFile (mod_pm, assembly_stream,
+                                         nullptr,  // FIXME: Correct?
```

Comment:
I look at the code, this seems correct.

Reply:
Looks right to me, too, but since I can't test the combination of OptiX/PTX and LLVM 7.0, I left this in so that if something breaks, we know I wasn't super sure.